src/server.c

```diff
@@ -5885,7 +5942,16 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "io_threaded_reads_processed:%lld\r\n"
             "io_threaded_writes_processed:%lld\r\n"
             "reply_buffer_shrinks:%lld\r\n"
-            "reply_buffer_expands:%lld\r\n",
+            "reply_buffer_expands:%lld\r\n"
+            "eventloop_cycles:%lld\r\n"
+            "eventloop_duration_sum:%lld\r\n"
+            "eventloop_duration_cmd_sum:%lld\r\n"
+            "eventloop_duration_io_read_sum:%lld\r\n"
+            "eventloop_duration_io_write_sum:%lld\r\n"
+            "eventloop_duration_aof_sum:%lld\r\n"
+            "eventloop_duration_cron_sum:%lld\r\n"
+            "instantaneous_eventloop_cycles:%lld\r\n"
+            "instantaneous_eventloop_duration_avg:%lld\r\n",
```

Comment:
Well, words like "instantaneous" and "t" are just old history code that I didn't bother to rename, and I just followed it to avoid too much change in code. As for naming and readability, maybe I am just used to the "code style" in Redis, I think some of your suggestions are unnecessary. However they do make sense, so if you insist, we can improve the code.

Reply:
I agree many of Jim's suggestions are not strictly necessary, but they do improve the code, so specifically in new code or places that are changed anyway, I think we should implement them (more variables that their name matches their usage, and the better matching type).

 Regarding "instantaneous" I think we must keep it and maybe find another way to make it slightly less awkward.
I see other metrics in this group all have units in their name ("per_sec" and "kbps")
So maybe we can drop the "average" part and make it:
`Instantaneous_eventloop_duration_usec` or `Instantaneous_eventloop_usec`.
And `Instantaneous_eventloop_cycles_per_sec`

Reply:
I suppse "usecs per seconds" is a rate. I almost feel like this should be `instantaneous_eventloop_usec_per_second`, which is the rate (or ratio) of time spend in the event loop per second.